.circleci/config.yml

```diff
@@ -20,6 +20,26 @@ aliases:
 
   - &run_cipublish
     - checkout
+    - run:
```

Comment:
If we take this approach, I would consider breaking these steps out into their own anchor. Something like `&update_machine_java`.